### DISCUSSION TITLE
My app store is suddenly missing, can someoen help?

### DISCUSSION THREAD

**Discussion by ovizii** [nocodb/nocodb#4590] [Q&A] [answered]
According to the manual from here: https://docs.nocodb.com/setup-and-usages/app-store/ it is supposed to look like this:

![image](https://user-images.githubusercontent.com/837902/205332069-a45f7b5d-f722-4eea-989b-64621d0bbfc6.png)

However on my self-hosted instance it looks like this:

![image](https://user-images.githubusercontent.com/837902/205332262-58861d6a-3b69-49db-aea2-074a128ddda6.png)

Any idea what is happening here?

**Comment by o1lab** [comment, marked as answer]
May be you are not the owner of the project ?

**Reply by ovizii** [reply]
Thanks, I also just found out that I might not be the super admin. Going through all my notes again and googling how to make myself the super admin. I'm the one who set up this instance and also the only user and I clearly remember having seen the app store there before, but I haven't used nocodb in a while. 

Let see what google yields :-)

**Comment by ovizii** [comment]
I can confirm that adding NC_ADMIN_EMAIL and NC_ADMIN_PASSWORD to an existing nocodb instance does indeed work. Using the same email address as my main user didn't work, so now I have separate access for the super admin account which works for me.
